### PR TITLE
Minor page fixes

### PR DESCRIPTION
**Pull request by somecodemonkey**
fixed some minor issues within pages and initial data
uses jqueryui to create dialogue popup to display full size of uploaded photos when they are clicked in the photo bar

### CODE REVIEW

File: app/views/Pages/myPage.html [hunk 1]
```diff
@@ -7,6 +7,11 @@
 	#{/if}
 	<div class = "page-data">
 	<div class ="page-title">${page.title}</div>
+	#{if page.admin != _user}
+		<div style="float: right">
+			<a class=button id = "fan${page.id}" href='#' onclick="return false;">*{text}*</a>
```
Comment (ProCynic) [inline app/views/Pages/myPage.html:12]: Might steal some of the like button js for this.

File: conf/initial-data.yml [hunk 2]
```diff
@@ -172,7 +172,7 @@ Page(cats):
     info:           >
                     I haz them!!
 
-Page(nachoCheese):
+Page(nachoCheese):
```
Comment (ProCynic) [inline conf/initial-data.yml:185]: Is there a reason you're adding a bunch of whitespace?

File: conf/initial-data.yml [hunk 3]
```diff
@@ -194,20 +194,15 @@ Post(cats post2):
                     CHEEZEBURGAHZ!!
                     
 Post(nachoCheese post1):
-    postType:       "PAGE"
-    title:          1
+    postedObj:      nachoCheese
     date:           2009-06-07
-    author:         jeff
+    owner:         jeff
```
Comment (ProCynic) [inline conf/initial-data.yml]: oops, I thought I had caught all of those.

**Comment by ProCynic** [comment]
Looks good.

File: conf/initial-data.yml (hunk 2 above)
Comment (somecodemonkey) [inline conf/initial-data.yml:185]: i had no idea i added white spaces

File: app/views/Pages/myPage.html (hunk 1 above)
Comment (somecodemonkey) [inline app/views/Pages/myPage.html:12]: will do

File: app/views/Pages/myPages.html [hunk 4]
```diff
@@ -1,6 +1,6 @@
 #{extends 'main.html' /} #{extends 'three_panel.html' /}
 <link type="text/css" rel="Stylesheet" href="/public/stylesheets/page.css" /> 
-<div id="contentArea">
+<div id="contentArea-AutoPage">
```
Comment (amshali) [inline app/views/Pages/myPages.html:3]: What is AutoPage?

File: app/views/main.html [hunk 5]
```diff
@@ -8,7 +8,7 @@
     <script src=@{'/public/javascripts/jquery.autoresize.js'}></script>
     <script type="text/javascript" src="/public/javascripts/jquery.ba-outside-events.js"></script>
     <script type="text/javascript" src="/public/javascripts/jquery.ba-outside-events.min.js"></script>
-    
+
```
Comment (amshali) [inline app/views/main.html:9]: ??

File: conf/initial-data.yml (hunk 3 above)
Comment (amshali) [inline conf/initial-data.yml]: Note to self: Refactoring problems....

File: app/views/main.html (hunk 5 above)
Comment (ProCynic) [inline app/views/main.html:9]: looks like whitespace removal

File: app/views/Pages/myPages.html (hunk 4 above)
Comment (somecodemonkey) [inline app/views/Pages/myPages.html:3]: Its in main.css and appeared to be used once we went back to the wide version of openbook

File: app/views/main.html (hunk 5 above)
Comment (somecodemonkey) [inline app/views/main.html:9]: Added a line then deleted so no change

File: app/views/Pages/myPages.html (hunk 4 above)
Comment (somecodemonkey) [inline app/views/Pages/myPages.html:3]: Its only as big as it needs to be while contentArea-bigpage sets a defined size

**Comment by somecodemonkey** [comment]
So why cant this get pulled in?

**Comment by amshali** [comment]
Merge problems. Can you try pulling and pushing again. I believe there is a merge conflict.

**Comment by somecodemonkey** [comment]
done

File: app/views/main.html (hunk 5 above)
Comment (ProCynic) [inline app/views/main.html:9]: The deleted line contains 4 spaces

**Pull request merged**